src/braket/ahs/pattern.py

```diff
@@ -24,3 +25,17 @@ def __init__(self, series: List[Number]):
     @property
     def series(self) -> List[Number]:
         return self._series
+
+    def discretize(self, resolution: Decimal) -> Pattern:
+        """Creates a discretized version of the pattern,
+        where each value is rounded to the nearest multiple
```

Comment:
Fine with either "closest" or "nearest," but usage should be consistent
```suggestion
        where each value is rounded to the closest multiple
```